packages/orchestration/src/orchestration-client.ts

```diff
@@ -140,13 +140,30 @@ export class OrchestrationClient {
       }
     });
 
-    const stream = OrchestrationStream._create(streamResponse, controller);
-    response.stream = stream
+    // Runs postProcessing of the response after the stream has been fully consumed.
+    function postProcessing(
```

Comment:
[q] Can you move this function to a separate private method? I imagine you would need to do something like 
`const postProcessWithContext = this.postProcessing.bind(this);` to use `this`